managed/ui/src/components/config/Security/KeyManagementConfigurationContainer.js

```diff
@@ -41,6 +41,13 @@ const mapDispatchToProps = (dispatch) => {
     setKMSConfig: (provider, body) => {
       return dispatch(createKMSProviderConfig(provider, body))
         .then?.((response) => {
+          if (response.error) {
+            const errorMessage =
+              response.payload?.response?.data?.error || response.payload.message;
```

Comment:
Any reason for OR here , does error message come in 2 different source ?

Reply:
Earlier implementation of toaster had a comment from Sergey, that this field may not contain a message in case of 500 or some scenarios like that, so we should also check for payload messages, However, I will check with @jitendra-12113 on this.

Reply:
`response.payload.message` holds a generic message in case of exception like 500.